man/en/sos.conf.5

```diff
@@ -3,7 +3,75 @@
 sos.conf \- sosreport configuration
 .SH DESCRIPTION
 .sp
-sosreport uses a configuration file at /etc/sos/sos.conf.
+sosreport uses a configuration file at /etc/sos/sos.conf, and there are
+subdirectories under /etc/sos that are used for specific purposes.
+
+Note that non-root users may override options set in /etc/sos/sos.conf by creating
+their own sos.conf under $HOME/.config/sos.
+
+The order in which options are loaded is as follows:
+
+  1. System configuration file at /etc/sos/sos.conf
+  2. User-specific configuration file at $HOME/.config/sos/sos.conf (for sos
+     components that support non-root)
+  3. In the case of running \fBsos report\fR, presets either automatically loaded
+     due to system configuration, or specified via \fB--add-preset\fR
```

Comment:
.. specified via `--add-preset` or just `--preset` (also)? The later worked to me as well.

Reply:
Whoops, that's a typo.